packages/flutter/test/painting/decoration_test.dart

```diff
@@ -152,8 +156,9 @@ void main() {
   });
 
   test('BoxDecorationImageListenerAsync', () {
-    FakeAsync().run((FakeAsync async) {
-      final ImageProvider imageProvider = AsyncTestImageProvider();
+    FakeAsync().run((FakeAsync async) async  {
+      final ui.Image image = await createTestImage(width: 10, height: 10);
```

Comment:
This should be moved a line above.

Reply:
(it does real async work and expects to complete before any other async operations it's enclosed by).

Reply:
Ah, yes! It was suppose to go there! Fixed.